test/XRP/reliable-submission-xrp-client.test.ts

```diff
@@ -132,4 +133,24 @@ describe('Reliable Submission XRP Client', function (): void {
     // THEN the function returns
     assert.deepEqual(result.hash, fakedTransactionResultValue.hash)
   })
+
+  it('authorizeSendingAccount - Returns when the transaction is validated', async function () {
```

Comment:
we seem to prefer treating the decorators as implementation details and testing the actual interface in `xrp-client-integration-test`. as such, i decided to just get the simplest test that makes sure things work here

Reply:
yep, this is exactly what I did with the other reliable submission methods in this file when I moved all the real tests to `CoreXrplClient` tests